src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    @Override
+    public String goWork() {
+        return "Truck started its work!";
+    }
+
+    @Override
+    public String stopWork() {
+        return "Truck stopped its work!";
+    }

```

Comment:
Similar to the `goWork` method, the message returned by the `stopWork` method should be more informative. Include the type of machine in the message to indicate it's a `Truck`, for example: `Truck has stopped its work!`.